setup.py

```diff
@@ -42,7 +42,7 @@ def read(fname):
     install_requires=[], # use pip -r requirements.txt instead
     entry_points={
         'console_scripts': [
-                'socorro = socorro.app.socorro_app:SocorroWelcomeApp.run'
+                'socorro = socorrolib.app.socorro_app:SocorroWelcomeApp.run'
```

Comment:
I haven't moved all apps into that namespace, merely the common ancestor code for the other apps that will be necessary for downstream apps to live in their own repos. It may not survive long as an entry point here, depending on the resolution of the app name shortcut shim compromise. We need it for now to keep automation and operations code working